### PR TITLE
OD-426 [Fix] Users will be able to see box-shadow in the LFD Agenda layout

### PR DESCRIPTION
**Pull request by YaroslavOvdii**
@sofiiakvasnevska 

## Issue
https://weboo.atlassian.net/browse/OD-426

## Description
Added a more specific CSS selector for the LFD item in the Agenda layout.

## Screenshots/screencasts
https://share.getcloudapp.com/Z4uozWPB

## Backward compatibility
This change is fully backward compatible.

## Reviewers
@upplabs-alex-levchenko @ivandevupp @AndrRyaz

### CODE REVIEW

File: scss/components/lfd/agenda.scss [hunk 1]
```diff
@@ -678,69 +678,71 @@
...
-                fontWeight: map-get($configuration, agendaTitleFontWeight),
-                fontStyle: map-get($configuration, agendaTitleItalic),
-                fontDecoration: map-get($configuration, agendaTitleUnderline)
-              ));
-
-              margin-top: map-get($configuration, agendaTitleMarginTop);
-              margin-bottom: map-get($configuration, agendaTitleMarginBottom);
-              color: map-get($configuration, agendaTitleFontColor);
+      .agenda-list-card-holder {
+        .agenda-list-item {
```
Comment (tonytlwu) [inline scss/components/lfd/agenda.scss:682]: Why indent everything when you can just do this?

```suggestion
      .agenda-list-card-holder .agenda-list-item {
```

I don't see any other declarations making use of the `.agenda-list-card-holder` selector.

**Review by tonytlwu — CHANGES_REQUESTED** [review]

File: scss/components/lfd/agenda.scss [hunk 2]
```diff
@@ -678,69 +678,71 @@
...
-                fontLineHeight: map-get($configuration, agendaTitleLineHeight),
-                fontWeight: map-get($configuration, agendaTitleFontWeight),
-                fontStyle: map-get($configuration, agendaTitleItalic),
-                fontDecoration: map-get($configuration, agendaTitleUnderline)
-              ));
-
-              margin-top: map-get($configuration, agendaTitleMarginTop);
-              margin-bottom: map-get($configuration, agendaTitleMarginBottom);
-              color: map-get($configuration, agendaTitleFontColor);
+      .agenda-list-card-holder {
```
Comment (tonytlwu) [inline scss/components/lfd/agenda.scss:681]:
```suggestion
```

File: scss/components/lfd/agenda.scss (hunk 1 above)
Comment (YaroslavOvdii) [reply scss/components/lfd/agenda.scss]: @tonytlwu You are right. Added suggestion to the code.

File: scss/components/lfd/agenda.scss (hunk 2 above)
Comment (YaroslavOvdii) [reply scss/components/lfd/agenda.scss]: @tonytlwu Fixed

**Review by tonytlwu — APPROVED** [review]
@sofiiakvasnevska This is ready for testing

Risk: High
Affected: LFD agenda items
Verify: Theme customisation is correctly applied

Squash and merge: Use PR title

**Pull request merged**